code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -124,6 +129,7 @@
 					on_reagent_change()
 					reagents.handle_reactions()
 					user.visible_message("<span class='notice'>[user] takes a blood sample from [target].</span>")
+					busy = 0

```

Comment:
Can you move this line up to be right after the previous ones? Something like:
```
busy = 1
if(!do_mob(user, target))
    busy = 0
    return
busy = 0
```
It's a bit hard to notice this one when it's all the way down here, and it shouldn't change the result at all.